website/pages/docs/drivers/java.mdx

```diff
@@ -97,6 +97,15 @@ task "web" {
 }
 ```
 
+## Capabilities
+
+| Feature | Implementation |
+| --- | --- |
+| Exec | false |
+| FSIsolation | FSIsolationNone  |
```

Comment:
On Linux the Java capabilities are slightly different: https://github.com/hashicorp/nomad/blob/v0.12.0/drivers/java/driver.go#L98-L101